docs/tutorial/quickstart.md

```diff
@@ -1,199 +1,272 @@
 
 Okay, now let's wire up the API URLs.  On to `tutorial/urls.py`...
 
-    from django.conf.urls import url, include
-    from rest_framework import routers
-    from tutorial.quickstart import views
+```python
+from django.conf.urls import url, include
+from rest_framework import routers
+from tutorial.quickstart import views
 
-    router = routers.DefaultRouter()
-    router.register(r'users', views.UserViewSet)
-    router.register(r'groups', views.GroupViewSet)
+router = routers.DefaultRouter()
+router.register(r'users', views.UserViewSet)
+router.register(r'groups', views.GroupViewSet)
 
-    # Wire up our API using automatic URL routing.
-    # Additionally, we include login URLs for the browsable API.
-    urlpatterns = [
-        url(r'^', include(router.urls)),
-        url(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework'))
-    ]
+# Wire up our API using automatic URL routing.
+# Additionally, we include login URLs for the browsable API.
+urlpatterns = [
+    url(r'^', include(router.urls)),
+    url(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework'))
+]
+```
 
-Because we're using viewsets instead of views, we can automatically generate the URL conf for our API, by simply registering the viewsets with a router class.
+Because we're using viewsets instead of views, we can automatically generate the
+URL conf for our API, by simply registering the viewsets with a router class.
 
-Again, if we need more control over the API URLs we can simply drop down to using regular class-based views, and writing the URL conf explicitly.
+Again, if we need more control over the API URLs we can simply drop down to
+using regular class-based views, and writing the URL conf explicitly.
 
-Finally, we're including default login and logout views for use with the browsable API.  That's optional, but useful if your API requires authentication and you want to use the browsable API.
+Finally, we're including default login and logout views for use with the
+browsable API.  That's optional, but useful if your API requires authentication
+and you want to use the browsable API.
 
 ## Settings
 
-We'd also like to set a few global settings.  We'd like to turn on pagination, and we want our API to only be accessible to admin users.  The settings module will be in `tutorial/settings.py`
+We'd also like to set a few global settings.  We'd like to turn on pagination,
+and we want our API to only be accessible to admin users.  The settings module
+will be in `tutorial/settings.py`
 
-    INSTALLED_APPS = (
-        ...
-        'rest_framework',
-    )
+```python
+INSTALLED_APPS = (
+    ...
+    'rest_framework',
+)
 
-    REST_FRAMEWORK = {
-        'DEFAULT_PERMISSION_CLASSES': ('rest_framework.permissions.IsAdminUser',),
-        'PAGE_SIZE': 10
-    }
+REST_FRAMEWORK = {
+    'DEFAULT_PERMISSION_CLASSES': (
+        'rest_framework.permissions.IsAdminUser',  # Note the trailing ',' character, this needs to be a tuple
```

Comment:
Does this setting still work okay if we changed it to a list instead, or does it need to be a tuple?